tools/bqtag/example/example.py

```diff
@@ -0,0 +1,125 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from bqtag import BQTableView
+
+if __name__ == "__main__":
+
```

Comment:
Remove blank line

Reply:
Changed to
```
if __name__ == "__main__":
    """
    - Update the BQ_PROJECT, CATALOG_PROJECT, BQ_DATASET and TAXONOMY_DISPLAY_NAME with relevant values.
    - Key file used is credentials.json. Please add the correct path or copy credentials.json to the example folder. 
    - SA should have relevant permissions to read Data Catalog Tags otherwise table creation will result in error.
    """
```